cfme/scripting/release.py

```diff
@@ -0,0 +1,211 @@
+)
+@click.option(
+    "--full",
+    "report_type",
+    flag_value="full",
+    help="Generates a full report with all PR description",
+)
+@click.option(
+    "--brief",
+    "report_type",
+    flag_value="brief",
+    default=True,
+    help="Generates report with PR, label, author, and title in 4 columns"
+)
+@click.option(
+    "--stats",
+    "report_type",
+    flag_value="stats",
+    help="Generates stats only report"
+)
+@click.option(
+    "--all",
+    "report_type",
+    flag_value="all",
+    help="Generates stats and brief report together"
+)
+@click.option(
+    "--links/--no-links",
+    default=False,
+    help="Include PR links in markdown"
+)
+@click.option(
+    "--format",
+    "tableformat",
+    default='github',
+    type=click.Choice(tabulate.tabulate_formats, case_sensitive=True),
+    help="The tablefmt option for python tabulate"
+)
+def main(tag, old_tag, report_type, links, tableformat):
+    """Script to assist in generating the release changelog
+
+    This script will generate a simple or full diff of PRs that are merged
+    and present the data in a way that is easy to copy/paste into an email
+    or git tag.
+    """
+
+    print(f"Report Includes: {old_tag} -> {tag}")
+
+    gh = github.Github(docker.gh_token)
+    repo = gh.get_repo(REPO_NAME)
+    release = repo.get_release(tag)
+    old_release = repo.get_release(old_tag)
+
+    prs = get_prs(release, old_release, gh)
+
+    if report_type in ("full", "brief", "all"):
+        # Print a markdown table of PR attributes, including description for full report type
+        report_data = []
+        for pr in prs:
+            pr_attrs = [
+                pr.number if not links else PR_LINK.format(pr=pr.number),
+                ', '.join(label.name for label in pr.labels if label.name in VALID_LABELS),
+                pr.user.login,
+                clean_commit(pr.title)
+            ]
+            if report_type == 'full':
+                pr_attrs.append(clean_body(pr.body))
+            report_data.append(pr_attrs)
+
+        print(tabulate.tabulate(report_data,
+                                headers=FULL_HEADERS if report_type == 'full' else HEADERS,
+                                tablefmt=tableformat))
+
+    elif report_type in ["stats", "all"]:
+        labels = defaultdict(int)
+        authors = defaultdict(int)
+
+        for pr in prs:
+            for label_name in [l.name for l in pr.labels if l.name not in IGNORED_LABELS]:
+                labels[label_name] += 1
+            authors[pr.user.login] += 1
+
+        # Label stats
+        print(tabulate.tabulate(sorted(labels.items(),
+                                       key=lambda item: item[1],
+                                       reverse=True),
+                                headers=["Label", "Number of PRs"],
+                                tablefmt=tableformat))
+
+        print('======================================')
+
+        # Author stats
```

Comment:
Anyway to get `Reviewer stats`? Definitely could/would be a separate PR

Reply:
Yes and yes :)